tests/testthat/helper-gcp.R

```diff
@@ -0,0 +1,34 @@
+skip_if_no_gcp <- function() {
+  skip_if(Sys.getenv("GCS_AUTH_FILE") == "")
+  skip_if_not_installed("googleCloudStorageR")
+  skip_if_offline()
+  skip_on_cran()
+}
+
+auth_gcp <- function() {
+  # the auth service email in this file needs GCP "Storage Admin" IAM role 
+  googleCloudStorageR::gcs_auth(Sys.getenv("GCS_AUTH_FILE"))
+}
+
+gcp_gcs_delete_bucket <- function(bucket){
+  # have to delete all objects in a bucket first
+  os <- googleCloudStorageR::gcs_list_objects(bucket, 
```

Comment:
Here is another instance where I would prefer to follow https://style.tidyverse.org/functions.html#long-lines-1. There are a couple more in this function.

Reply:
This code block was pulled into latest CRAN release of googleCloudStorageR 0.7.0